src/orion/algo/pbt/pbt.py

```diff
@@ -802,6 +823,7 @@ def fork(self, new_trial):
             identical working_dir would imply that different trials have identical parameters.
 
         """
+        # BUG: Here getting the working_dir property is causing an issue.
```

Comment:
Is it related to my comment here? https://github.com/Epistimio/orion/pull/833/files#r834557496

Reply:
Not quite. This was before I copied over the experiment in `_copy_status_and_results` of `primary_algo.py`.

Fixed in 0a2a2494da470b591a6bf06584f1fe9f750c881a